dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/JdbcSelectTask.java

```diff
@@ -35,6 +37,10 @@ public class JdbcSelectTask extends AbstractJdbcTask<Summary> {
 
   @Nonnull private final TaskCategory taskCategory;
 
+  private boolean shouldCalculateQueryLogDates;
```

Comment:
I introduced this variable to differentiate the cases when we should not consider dates as first or last entry. For now it is used in SnowflakeLogsConnector, where we run time series task with frequency of 1 day, so they consider aggregated values. Because that tasks are part of logs connector work, we should filter out that dates. More comment [here](https://github.com/google/dwh-migration-tools/pull/560/files#r1777341288)